source/connect.txt

```diff
@@ -123,3 +141,88 @@ For details, see :ref:`comp-command-line-connect`.
    Favorite Connections </connect/favorite-connections>
    Required Access </connect/required-access>
    Disconnect </connect/disconnect>
+
+Connection String Examples
+--------------------------
+
+The following examples highlight some commonly misunderstand advanced connection
+string options, particularly when connecting to Compass. For information on 
+all connection string options, see :ref:`connections-connection-options`.  
+
+TLS Options
+~~~~~~~~~~~
+
+The following connection string enables TLS using the ``tls`` URI option. When
+specifying TLS/SSL certificates in your connection string, you must specify the ``tlsCAFile``
+as the path to the Certificate Authority file, URL-encoded, and the ``tlsCertificateKeyFile``
+as the path to the combined server certificate and private key file, URL-encoded.
+
+.. code-block:: 
+
+   mongodb://myDatabaseUser:D1fficultP%40ssw0rd@db0.example.com/?directConnection=true&tls=true&tlsCAFile=path/to/rootCA.crt&tlsCertificateKeyFile=path/to/server_certificate.pem
+
+To enable SSL, use the equivalent ``ssl=true`` option in place of ``tls=true``. 
+
+X.509 Authentication
+````````````````````
+
+The following example demonstrates how to specify X.509 authentication in your connection string
+using the :urioption:`authMechanism` connection string option:
+
+.. code-block:: 
+
+   mongodb://myDatabaseUser:D1fficultP%40ssw0rd@db1.example.com/?authMechanism=MONGODB-X509&tls=true&tlsCertificateKeyFile=%2Fpath%2Fto%2Fclient_certificate.pem
+
+LDAP
+~~~~
+
+The following example specifies LDAP authentication. It sets :urioption:`authMechanism` to ``plain`` 
+to specify the use of the LDAP authentication mechanism, and sets ``authSource`` to ``$external``
+to indicate that the authentication database is ``$external``.
+
+.. code-block:: 
+
+   mongodb://myDatabaseUser:D1fficultP%40ssw0rd@db2.example.com/?authMechanism=PLAIN&authSource=$external
+
+Kerberos
+~~~~~~~~
+
+The following example specifies Kerberos authentication. It sets :urioption:`authMechanism` to
+``GSSAPI`` to use the Kerberos authentication mechanism, and sets ``authSource`` to ``$external``
+to indicate that the authentication database is ``$external``.
+
+.. code-block:: 
+
+   mongodb://myDatabaseUser:D1fficultP%40ssw0rd@db3.example.com/?authMechanism=GSSAPI&authSource=$external
```

Comment:
There are some situations in which you would use a password with Kerberos, but those are mostly for some very specific and uncommon use cases